CRM/Utils/System/Standalone.php

```diff
@@ -259,6 +264,18 @@ public function theme(&$content, $print = FALSE, $maintenance = FALSE) {
       echo implode('', $smarty->_tpl_vars['pageHTMLHead']);
     }
 
+    // Show the breadcrumb
+    if (!empty(\Civi::$statics[__CLASS__]['breadcrumb'])) {
+      print '<nav aria-label="' . ts('Breadcrumb', ['escape' => 'js']) . '" class="breadcrumb"><ol>';
```

Comment:
Aside: I don't entirely understanding [this encoding](https://github.com/civicrm/civicrm-core/blob/83adaedfb9b754d9915622b85f5f01cdf5f127e7/CRM/Core/I18n.php#L58). Should it be  `htmlentities(ts('Breadcrumb'))` or `htmlspecialchars(ts('Breadcrumb'))`?

Reply:
It's an odd convention (and not mine) and probably overkill, but it works well for encoding strings inside HTML attributes, and widely used in core.

I compared the output of the 3 methods:

```
htmlentities = Bread&#039;cr\&quot;mb
htmlspecialchars = Bread&#039;cr\&quot;mb
json_encode = Bread\u0027cr\\\u0022mb
```

Code:

```php
<?php
$text = 'Bread\'cr\\"mb';
echo 'htmlentities = ' . htmlentities($text, ENT_QUOTES) . "\n";
echo 'htmlspecialchars = ' . htmlspecialchars($text, ENT_QUOTES) . "\n";
echo 'json_encode = ' . substr(json_encode($text, JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE | JSON_HEX_TAG | JSON_HEX_AMP | JSON_HEX_APOS | JSON_HEX_QUOT), 1, -1) . "\n";
```

Reply:
I don't have any opinion on this, except that if we change that convention, we should document/standardize on it.

By "works well", I meant mostly that we've had issues with this in the past (French uses the single-quote a lot), and we haven't run into issues recently.

Would it make sense to have `escape => css' or just `htmlspecialchars` because that's more explicit?